pep-0594.rst

```diff
@@ -476,7 +476,7 @@ mailcap
 
 The `mailcap <https://docs.python.org/3/library/mailcap.html>`__ package
 reads *mail capability* files to assist in handling a file attachment in
-an email. In most modern OSs the email client itself handles reacting to
+an email. In most modern OSes the email client itself handles reacting to
```

Comment:
Thanks! I had never seen `OSs` and assumed it was just a typo. Let's just write it out so we don't have to worry about the spelling:
```suggestion
an email. In most modern operating systems the email client itself handles reacting to
```